### PR TITLE
Strapi - Case Study

### PR DESCRIPTION
**Pull request by NehGuk**
Could you please let me know if my case study is heading in the right direction? I'm pretty much done but intend to add a final chapter with a quick demonstration on how to set up a self-hosted API with PostgreSQL.

### CODE REVIEW

File: strapi.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+- Strapi may not be the ideal platform for projects primarily written in Typescript. There surely are external tools and resources that allow developers to integrate Typescript in their Strapi projects. However, the fact that Typescript has not been fully incorporated into the Strapi world yet may be a relevant limitation to some developers.
+- Strapi does not have a native hosting environment yet, although they have recently announced StrapiCloud, as a tool which will fill this gap in the future.
+- Strapi is rapidly evolving and that means it is frequently updated. Some of these updates may be disruptive, as they implement breaking changes that may compromise some projects.
+
+## 6. Comparison
+
+_Compare the product to other products in the same category, if applicable._
```
Comment (Fermain) [inline strapi.md:64]:
```suggestion
```

**Review by Fermain — CHANGES_REQUESTED** [review]
This is going in the right direction, but I have one major intervention. I am instructing students not to use first person directly, and switch to `The author...` where that isn't possible. I don't think that will be a huge issue for you and its a matter of consistency more than anything else.

**Comment by Fermain** [comment]
Please mark this as a draft

**Comment by NehGuk** [comment]
Thank you for the feedback, Oliver. I have edited out the use of the first person. I have also included the missing final chapters.

**Review by Fermain — APPROVED** [review]
A detailed, well researched case study that provides a well rounded introduction to Strapi. This is a tool that many of our students and teachers use, so this case study will be of great use to interested parties in future.

The review process shows that you are able to take on constructive feedback and implement this in your work without issues. Given your background in technical journalism, this was not a surprise to me :)

Well done and thanks for this great quality contribution!

**Pull request merged**